django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -144,17 +166,18 @@
 }
 
 #changelist-filter a {
+    display: block;
     color: #999;
 }
 
 #changelist-filter a:hover {
-    color: #036;
+    color: #036 !important;
```

Comment:
Right, this `!important` isn't necessary here. I can work on that ticket later btw -- I explored admin a lot :)